### PR TITLE
Moved GThreadNames to CaptureData and changed std::unordered_map to absl::flat_hash_map

### PR DESCRIPTION
**Pull request by florian-kuebler**

### CODE REVIEW

File: OrbitCore/CaptureData.h [hunk 1]
```diff
@@ -54,7 +54,26 @@ class CaptureData {
...
-  orbit_client_protos::LinuxAddressInfo* GetAddressInfo(uint64_t address);
+  [[nodiscard]] orbit_client_protos::LinuxAddressInfo* GetAddressInfo(
+      uint64_t address);
+
+  [[nodiscard]] const absl::flat_hash_map<int32_t, std::string>&
+  thread_names() {
+    return thread_names_;
+  }
+
+  [[nodiscard]] const std::string& thread_name(int32_t thread_id) {
```
Comment (dimitry-) [inline OrbitCore/CaptureData.h:65]: this is not trivial getter - it should probably be GetThreadName

File: OrbitCore/CaptureData.h [hunk 2]
```diff
@@ -54,7 +54,26 @@ class CaptureData {
     address_infos_ = std::move(address_infos);
   }
 
-  orbit_client_protos::LinuxAddressInfo* GetAddressInfo(uint64_t address);
+  [[nodiscard]] orbit_client_protos::LinuxAddressInfo* GetAddressInfo(
+      uint64_t address);
+
+  [[nodiscard]] const absl::flat_hash_map<int32_t, std::string>&
+  thread_names() {
```
Comment (dpallotti) [inline OrbitCore/CaptureData.h:61]: Can this method be `const`?

File: OrbitCore/CaptureData.h [hunk 3]
```diff
@@ -54,7 +54,26 @@ class CaptureData {
...
+    return thread_names_;
+  }
+
+  [[nodiscard]] const std::string& GetThreadName(int32_t thread_id) {
```
Comment (dpallotti) [inline OrbitCore/CaptureData.h:65]: Can this method be `const`?

File: OrbitCore/CaptureData.h (hunk 1 above)
Comment (florian-kuebler) [reply OrbitCore/CaptureData.h]: Done. Thanks for the hint

File: OrbitGl/GpuTrack.cpp [hunk 4]
```diff
@@ -217,7 +217,7 @@ std::string GpuTrack::GetSwQueueTooltip(const TimerInfo& timer_info) const {
       "<br/>"
       "<b>Submitted from thread:</b> %s [%d]<br/>"
       "<b>Time:</b> %s",
-      Capture::GThreadNames[timer_info.thread_id()].c_str(),
+      Capture::capture_data_.GetThreadName(timer_info.thread_id()).c_str(),
```
Comment (dpallotti) [inline OrbitGl/GpuTrack.cpp:220]: Previously like this but you don't need `c_str()` in `StrFormat` (also below).

Comment (florian-kuebler) [reply OrbitGl/GpuTrack.cpp]: Done. Thanks

File: OrbitCore/CaptureData.h (hunk 3 above)
Comment (florian-kuebler) [reply OrbitCore/CaptureData.h]: Made all access methods in CaptureData const where possible.

**Pull request merged**